### DISCUSSION TITLE
win.text stuff

### DISCUSSION THREAD

**Discussion by DvdSW** [ikemen-engine/Ikemen-GO#871] [Feature Request] [answered]
if is possible, have **win.anim** and **win.text** working together? (if you use animation it automatically disable win.text ) also maybe in the future can we have something like **win.text.start.x**/ **win.text.start.y**?

**Comment by K4thos** [comment, marked as answer]
no (at least not implemented by me).

Win support 32 layers of animated bg elements. Additional animated element sounds completely useless to me. It would add complexity to code and create backward compatibility problems (you can bet that there are lifebars with both text and anim elements added, with only one meant to work - leftovers are to be expected if they are ignored by mugen).

start.x / y are not needed when you already can declare offset. If you mean position changes in a style of combo messages done in-engine than that's excessive, imo, when you can already animate it with `anim` element. It would require tons of additional code for something that barely anyone would use.
